components/role-mgt/org.wso2.carbon.identity.role.mgt.core/src/main/java/org/wso2/carbon/identity/role/mgt/core/RoleConstants.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.role.mgt.core;
+
+/**
+ * Role constants.
+ */
+public class RoleConstants {
+
+    private RoleConstants() {
+
+    }
+
+    public static final String DISABLED = "Disabled";
+
+    // DB product names.
+    public static final String H2 = "H2";
+    public static final String MY_SQL = "MySQL";
+    public static final String ORACLE = "Oracle";
+    public static final String MICROSOFT = "Microsoft SQL Server";
+    public static final String POSTGRE_SQL = "PostgreSQL";
+    public static final String DB2 = "DB2";
+    public static final String INFORMIX = "INFORMIX";
+
+    // Named query fields.
+    public static final String OFFSET = "OFFSET";
+    public static final String LIMIT = "LIMIT";
+    public static final String ZERO_BASED_START_INDEX = "ZERO_BASED_START_INDEX";
+    public static final String ONE_BASED_START_INDEX = "ONE_BASED_START_INDEX";
+    public static final String END_INDEX = "END_INDEX";
+    public static final String WILDCARD_CHARACTER = "*";
+
+
+    // Group related constants.
+    public static final String ID_URI = "urn:ietf:params:scim:schemas:core:2.0:id";
+
+    /**
+     * Grouping of constants related to database table names.
+     */
+    public static class RoleTableColumns {
+
+        public static final String UM_ROLE_NAME = "UM_ROLE_NAME";
+        public static final String UM_TENANT_ID = "UM_TENANT_ID";
+        public static final String NEW_UM_ROLE_NAME = "NEW_UM_ROLE_NAME";
+        public static final String UM_USER_NAME = "UM_USER_NAME";
+        public static final String UM_GROUP_NAME = "UM_GROUP_NAME";
+        public static final String UM_DOMAIN_NAME = "UM_DOMAIN_NAME";
+
+        public static final String TENANT_ID = "TENANT_ID";
+        public static final String ATTR_NAME = "ATTR_NAME";
+        public static final String ATTR_VALUE = "ATTR_VALUE";
+        public static final String ROLE_NAME = "ROLE_NAME";
+        public static final String NEW_ROLE_NAME = "NEW_ROLE_NAME";
+
+        private RoleTableColumns() {
+
+        }
+    }
+
+    /**
+     * Error message enums.
+     * <p>
+     * Error codes for server errors start with 75 and client errors start with 70.
+     */
+    public enum Error {
+
+        UNEXPECTED_SERVER_ERROR("75001"),
```

Comment:
shall we follow the error code standard defined in "[Architecture] WSO2 Identity Server REST API Error Response Standardization" ?

Reply:
Changed the error codes accordingly and updated the PR.